exporter/stackdriverexporter/README.md

```diff
@@ -1,3 +1,47 @@
 # Stackdriver Exporter
 
-To be added.
\ No newline at end of file
+This exporter can be used to send metrics to Stackdriver.
+
+The following configuration options are supported:
+
+- `project` (required): ProjectID is the identifier of the Stackdriver.
```

Comment:
I was able to use `stackdriver` exporter with no config parameters at all, so none of the parameters in the list are required.

Specifically for `project` it's worth mentioning that the project will be automatically detected in the GCP environment unless explicitly overridden via this parameter.

Reply:
@nilebox Thanks for helping. fixed at lastest commit.

Reply:
Thanks! Also need to rephrase the description, see new comment.